src/model/include/iDynTree/ModelTestUtils.h

```diff
@@ -167,6 +235,31 @@ inline Model getRandomChain(unsigned int nrOfJoints, size_t nrOfAdditionalFrames
     return model;
 }
 
+// Backward compatibility overloads
+inline Model getRandomModel(unsigned int nrOfJoints, size_t nrOfAdditionalFrames, bool onlyRevoluteJoints, bool includeRevoluteJointsSO2 = false)
```

Comment:
```suggestion
inline Model getRandomModel(unsigned int nrOfJoints, size_t nrOfAdditionalFrames, bool onlyRevoluteJoints)
```

for back compatibility, we just need to provide the old API, not add a new attribute to a deprecated function.